packages/react-components/react-tree/stories/Tree/TreeWithoutInlineStyle.stories.tsx

```diff
@@ -0,0 +1,68 @@
+import * as React from 'react';
```

Comment:
This story will need a nice description, too 😉

Reply:
All of them need 🙈, I'll worry about documenting once I hit unstable